src/Features/Admin/Users/index.js

```diff
@@ -1,25 +1,43 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
+import { useDispatch } from 'react-redux';
 import { useHistory } from 'react-router';
 
 import CommonButton from '../../../components/Button/Button';
 import CustomTable from '../../../components/CustomTable';
-import { userList } from '../../../Mock/UserList';
 import RouteNames from '../../../routes/RouteNames';
+import { FetchUsers } from '../Common Requests/request';
+import { deleteUserById } from './actions';
 import { UsersTitleContainer, UsersTitle } from './style';
+
 function UsersList() {
-  const header = ['Id', 'Name', 'Email', 'Contact', 'Edit'];
+  const [users, setUsers] = useState([]);
+
+  const usersData = FetchUsers('user');
```

Comment:
Are we not showing loader while API fetches data and errrors .. please use documentation to see how we can utilize built in loading and error flags to cater loading and error